pyrdp/player/LiveTab.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, parent: QWidget = None):
         self.layers = layers
         self.rdpWidget = rdpWidget
         self.fileSystem = FileSystem()
-        self.eventHandler = LiveEventHandler(self.widget, self.text, self.log, self.fileSystem, self.layers.player)
+        self.eventHandler = LiveEventHandler(self.widget, self.text, self.log, self.fileSystem, self.layers.player, self.connectionRenameTab, self)
```

Comment:
It seems a bit strange to pass signals around to me. In C++ signals are just methods and not objects, so this wouldn't be possible to do trivially. Consider moving the signals to the live event handler directly and exposing the signals through other signals or just doing something like `self.connectionRenameTab = self.eventHandler.connectionRenameTab`.

It seems like you can even connect signals to other signals just like in C++ so this should not involve too much duplication: https://groups.google.com/d/msg/pyside/xlRLJfGmGYM/3uKYkf98jckJ

It makes the class more self-contained and easier to understand IMO.

Reply:
PyQt signals needs to be defined on a class of type QObject, and the LiveEventHandler isn't one which is why I did it this way. I'll look into it

Reply:
Done.